provider/kubernetes/kubernetes.go

```diff
@@ -308,6 +310,56 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 				}
 			}
 		}
+
+		for _, t := range i.Spec.TLS {
+			tlsSecret, exists, err := k8sClient.GetSecret(i.Namespace, t.SecretName)
+			if err != nil {
+				log.Errorf("Unable to retrieve secret %s/%s: %s", i.Namespace, t.SecretName, err)
+				continue
+			}
+			if !exists {
+				log.Warnf("Unable to find secret %s/%s", i.Namespace, t.SecretName)
```

Comment:
_find_ here and _retrieve_ above are very similar. Can we say something like _Secret %s/%s does not exist_?